### DISCUSSION TITLE
Is sycl::ctz ready ?

### DISCUSSION THREAD

**Discussion by zjin-lcf** [intel/llvm#4450] [Q&A]
With the latest build, the compiler shows the message:

error: no member named 'ctz' in namespace 'sycl'; did you mean 'clz'?

Thanks

**Comment by AlexeySachkov** [comment]
Hi @zjin-lcf,

I can find this built-in in the spec (Section [4.17.6. Integer functions](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_integer_functions)). So, looking at the error message you posted it seems to me that we have a bug in our implementation, let me convert this discussion into an issue

**Reply by zjin-lcf** [reply]
Okay. Thanks for your update.

**Reply by AlexeySachkov** [reply]
See #4451
